configure.ac

```diff
@@ -1053,6 +1061,11 @@ if test x$enable_exult_studio = xyes; then
 		echo "Try again, either with Gtk-3.16 or newer, or with --disable-exult-studio"
 		exit 1
 	fi
+	if test x$have_icu = xno; then
+		echo "Umm, but we don't have any Icu stuff."
+		echo "Try again, either with Icu, or with --disable-exult-studio"
```

Comment:
Maybe these two should say libicu instead of Icu? Or maybe ICU, all caps, like they do in their website?

Reply:
I just copied the check from GTK+ just above, and it uses `Gtk+` not `GTK+`. Should we change both to all caps ?

Reply:
Might as well.